flocker/restapi/_infrastructure.py

```diff
@@ -223,7 +223,7 @@ def got_result(result):
     return deco
 
 
-def user_documentation(doc, examples=None):
+def user_documentation(doc, examples=None, header=None):
```

Comment:
unfortunately there is code that expects the 2nd parameter to be "examples", so header has to be 3rd. Therefore it needs to have this default value.